plugins/catalog-backend-module-azure/config.d.ts

```diff
@@ -34,6 +34,10 @@ export interface Config {
           /**
            * (Required) Your organization slug.
            */
+          apiVersion?: string;
+          /**
+           * (Optional) azure devops API version.
```

Comment:
This should be above the property, I think you ight have inverted this by mistake.